Assignment3/Assignment3.py

```diff
@@ -0,0 +1,185 @@
+import sys
+
+from pyspark import keyword_only
+from pyspark.ml import Pipeline, Transformer
+from pyspark.ml.param.shared import HasInputCols, HasOutputCol
+from pyspark.ml.util import DefaultParamsReadable, DefaultParamsWritable
+from pyspark.sql import SparkSession, Window
+from pyspark.sql.functions import col, sum
+
+
+def main():
+    # Setup Spark
+    spark = SparkSession.builder.master("local[*]").getOrCreate()
+
+    # put in sql database username here
+    username = ""
+    # put in sql database password here
+    pword = ""
```

Comment:
Not necessary, but could put an echo in the sh script letting user know to place un and pwd in the py file.